cmd/createtree/main.go

```diff
@@ -147,9 +147,6 @@ func newPK(opts *createOpts) (*any.Any, error) {
 			return nil, fmt.Errorf("error reading PEM key file at %v: %v", path, err)
 		}
```

Comment:
The code above that checks for existence of the PEM file needs to be deleted as well. This only works if the createtree command is run on the same machine as the Trillian server, since the file path is resolved by the server, not the client.